pod-identity/README.md

```diff
@@ -0,0 +1,48 @@
+# Use Pod Identity with EKS for Domino Workloads based on Domino Service Accounts
+
+[EKS Pod Identities](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html) allows a K8s pod assume an AWS role based on the ServiceAccount associated with the Pod
+
+Domino Workloads have a dynamically generated Service Accounts. To use Pod Identities we need to use Static Service Accounts
+
+Follow these steps:
+
+1. Domino Service Accounts can be created based on the following [docs](https://docs.dominodatalab.com/en/latest/admin_guide/6921e5/domino-service-accounts/). 
```

Comment:
Do `ServiceAccount` objects get created (either in platform or compute) whenever someone creates a Domino Service Account?

Reply:
No. Domino Service Accounts are actually user accounts in Mongo and Keycloak. They just don't have a pwd or any ways to login via the UI.

Reply:
Also I left a comment in the description
```This documentation is created just for FINRA. They wanted a way to attach Domino Service Accounts to IAM roles. They cannot use IRSA because every time they do a Blue/Green deployment they have to update IAM Trust Policies and the EKS team does not own IAM```